synapse/http/server.py

```diff
@@ -231,9 +233,6 @@ def _async_render(self, request):
             self._send_response(request, 200, {})
             return
 
-        request_metrics = RequestMetrics()
-        request_metrics.start(self.clock)
```

Comment:
I wouldn't remove these metrics, since the outer metrics won't know what callback is being invoked,